frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -467,6 +472,12 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 setShouldStripQueryParams: (_, { shouldStripQueryParams }) => shouldStripQueryParams,
             },
         ],
+        conversionGoal: [
+            null as WebAnalyticsConversionGoal | null,
```

Comment:
Going with the above suggestion would make this null redundant
```suggestion
            null as WebAnalyticsConversionGoal,
```